pages/404.tsx

```diff
@@ -9,8 +9,7 @@ export default function NotFound() {
       <NextSeo title="Page not found" />
       <div className="justify-center align-center text-center">
         <NextImage
-          // layout="fill"
-          objectFit="cover"
+          object-fit="cover"
```

Comment:
i have left a comment on the 404 page that i will need to resolve before merging.